### PR TITLE
Bug Fix: unhighlighting inexistent placeguides

### PR DESCRIPTION
**Pull request by bori00**
#### Context
When a user tries to highlight a placeguide, any previously highlighted guide will be "unhighlighted".

#### Issue
If previously highlighted guide is not on the current map area anymore, the corresponding infobox div and PlaceGuideOnMap object do not exist anymore. Any actions that should be performed on these object will cause a crash. 

#### Solution
To avoid this crash, I simply need to check if the corresponding objects still exist.

### CODE REVIEW

**Comment by bori00** [comment]
Please remark that some removals in the code refer to some unnecessary duplications in the code which may have been caused by a poorly resolved merge conflict.

File: pom.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
                     <!-- TODO: set project ID. -->
                     <deploy.projectId>denniswillie-step-2020</deploy.projectId>
                     <deploy.version>1</deploy.version>
+                    <port>8181</port>
```
Comment (cgollner) [inline pom.xml]: Seems like this change is unrelated. Can we leave it out?

File: src/main/webapp/placeGuides/PlaceGuideOnList.js [hunk 2]
```diff
@@ -418,8 +418,12 @@ class PlaceGuideOnList {
   static close(placeGuideId) {
     const divId = 'placeGuideOnList-' + '{' + placeGuideId + '}';
     const placeGuideDiv = document.getElementById(divId);
-    placeGuideDiv.querySelectorAll('.folded-placeGuide')[0].style.display = 'block';
-    placeGuideDiv.querySelectorAll('.card-placeGuide')[0].style.display = 'none';
-    placeGuideDiv.style.removeProperty('padding');
+    if (placeGuideDiv != null) {
```
Comment (cgollner) [inline src/main/webapp/placeGuides/PlaceGuideOnList.js]: Should we compare to `undefined` instead? Or is it intentional?

**Review by cgollner — CHANGES_REQUESTED** [review]

File: src/main/webapp/placeGuides/PlaceGuideOnList.js (hunk 2 above)
Comment (bori00) [reply src/main/webapp/placeGuides/PlaceGuideOnList.js]: It was intentional, because document.getElementById() returns null if the element is not found.

File: pom.xml (hunk 1 above)
Comment (bori00) [reply pom.xml]: Sure. I removed it.

**Review by cgollner — APPROVED** [review]
